JustSaying/JustSayingFluently.cs

```diff
@@ -187,9 +187,20 @@ public ISubscriberIntoQueue WithSqsPointToPointSubscriber()
 
         #region Implementation of Queue Subscription
 
-        public IFluentSubscription IntoQueue(string queuename)
+        public IFluentSubscription IntoQueueNamed(string queueName)
         {
-            _subscriptionConfig.BaseQueueName = queuename;
+            if (string.IsNullOrWhiteSpace(queueName))
+            {
+                throw new ArgumentNullException(nameof(queueName));

```

Comment:
Maybe include in the exception message something to saying use ```.IntoDefaultQueue()``` if that was their intent. They might have found a compile error after upgrading and done a wholesale find & replace of all ```IntoQueue```-> ```IntoQueueNamed```